src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -470,8 +471,21 @@ def create(
             spark_version = ws.clusters.select_spark_version(latest=True)
         if single_node:
             kwargs["num_workers"] = 0
-            kwargs["spark_conf"] = {"spark.databricks.cluster.profile": "singleNode", "spark.master": "local[*]"}
+            if spn_example:
+                kwargs["spark_conf"] = {
```

Comment:
Add spark_conf argument instead of spn example, handle merging with single-node cases